Src/BootCamp.Chapter/Program.cs

```diff
@@ -9,6 +9,22 @@ class Program
     {
         static void Main(string[] args)
         {
+            // Variables
+            string nameSurname = " ";
```

Comment:
don't declare all variables in one place- declare them as close as possible to where they are used